chat-plugins/info.js

```diff
@@ -14,7 +14,7 @@
 
 const RESULTS_MAX_LENGTH = 10;
 
-let commands = exports.commands = {
+let commands = exports.commands = { // eslint-disable-line no-unused-vars
```

Comment:
I'd prefer that too.

However, this one should probably remain https://github.com/Zarel/Pokemon-Showdown/pull/2256/files#diff-4e71f9f8fa7dc574ad692d530284d882R25
As it can be referenced by the ``/eval`` command.